index.html

```diff
@@ -0,0 +1,772 @@
+            <img alt="image"
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/biggst-ev-motor-electric-car-future-diesel-factory-production-future-PSA-world-480x270.png">
+            <div>
+                <h4><a class="section2_links" href="">It’s happening! World’s biggest diesel factory pivots to electric
+                    motors</a></h4>
+                <div>
+                    <a class="special_text" href="#">Matthew Beedham</a>
+                    <p class="special_text">· 6 hours ago</p>
+                </div>
+            </div>
+        </div>
+        <div class="section2_items">
+            <img alt="image"
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/1-copy-51-516x270.jpg">
+            <div>
+                <h4><a class="section2_links" href="">This nearly inexplicable exoplanet has the density of cotton candy</a></h4>
+                <div>
+                    <a class="special_text" href="#">The Cosmic Companion</a>
+                    <p class="special_text">· 6 hours ago</p>
+                </div>
+            </div>
+        </div>
+        <div class="section2_items">
+            <img alt="image"
+                src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/biden-electric-cars-future-us-state-vehicles-car-EV-480x270.jpg">
+            <div>
+                <h4><a class="section2_links" href="">The future of US government vehicles is electric</a></h4>
+                <div>
+                    <a class="special_text" href="#">Matthew Beedham</a>
+                    <p class="special_text">· 7 hours ago</p>
+                </div>
+            </div>
+        </div>
+        <div class="section2_items">
+          <img alt="image"
+              src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2021/01/Signal-hed-513x270.jpg">
+          <div>
+              <h4><a class="section2_links" href="">Signal is drama-free for now, but it should prepare for the worst</a></h4>
+              <div>
+                  <a class="special_text" href="#">Abhimanyu Ghoshal</a>
+                  <p class="special_text">· 8 hours ago</p>
+              </div>
+          </div>
+
+      </div>
+    </section>-->
+
+    <section class="ltst-fndg">
+      <a href="#">
+        <h2>Latest funding rounds on index.co ></h2>
+      </a>
+      <div></div>
+      <nav class="more">
+        <a href="#" class="more">Add info</a>
+        <p class="hide">|</p>
+        <a id="border_LA" href="#" class="more">Get updates by email</a>
+        <p class="hide">|</p>
+        <a href="#" class="more">See more</a>
+      </nav>
+      <div class="founding-cnt">
```

Comment:
This should be an `article` tag. Same applies to all the other 3 remaining 'cards'.